test/functional/fixtures/shell-test.c

```diff
@@ -4,7 +4,12 @@
 #include <stdio.h>
 #include <string.h>
 #include <stdint.h>
+#ifdef WIN32
+#include <Windows.h>
+#define usleep(usecs) Sleep(usecs/1000)
```

Comment:
I think this should only be done `#ifdef _MSC_VER` . Maybe that's why the terminal tests fail on the mingw builds.

Reply:
Yes, I agree that it should be `_MSC_VER`, but I don't that it's causing any problems because the MinGW builds should be able to use all the Windows headers and functions.